mxcube3/core/components/user/usermanager.py

```diff
@@ -22,7 +25,7 @@ def get_observers(self):
         return [
             user
             for user in User.query.all()
-            if (not user.in_control and user.is_authenticated)
+            if ((not user.in_control) and user.is_authenticated and user.is_active)
```

Comment:
not entirely sure about this:

```suggestion
            if ((not user.in_control) and user.is_authenticated and user.active)
```

Reply:
Well you could use both, active is the attribute and is_active is the property pointing to the same. I prefer to use the attribute in this case.